src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1,2746 +1,2574 @@
-//***************************************************************************
-std::string UsgsAstroLsSensorModel::getImageIdentifier() const {
-  return m_imageIdentifier;
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::setImageIdentifier
-//***************************************************************************
-void UsgsAstroLsSensorModel::setImageIdentifier(const std::string& imageId,
-                                                csm::WarningList* warnings) {
-  // Image id should include the suffix without the path name
-  m_imageIdentifier = imageId;
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::getSensorIdentifier
-//***************************************************************************
-std::string UsgsAstroLsSensorModel::getSensorIdentifier() const {
-  return m_sensorIdentifier;
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::getPlatformIdentifier
-//***************************************************************************
-std::string UsgsAstroLsSensorModel::getPlatformIdentifier() const {
-  return m_platformIdentifier;
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::setReferencePoint
-//***************************************************************************
-void UsgsAstroLsSensorModel::setReferencePoint(
-    const csm::EcefCoord& ground_pt) {
-  m_referencePointXyz = ground_pt;
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::getReferencePoint
-//***************************************************************************
-csm::EcefCoord UsgsAstroLsSensorModel::getReferencePoint() const {
-  // Return ground point at image center
-  return m_referencePointXyz;
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::getSensorModelName
-//***************************************************************************
-std::string UsgsAstroLsSensorModel::getModelName() const {
-  return UsgsAstroLsSensorModel::_SENSOR_MODEL_NAME;
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::getImageStart
-//***************************************************************************
-csm::ImageCoord UsgsAstroLsSensorModel::getImageStart() const {
-  return csm::ImageCoord(0.0, 0.0);
-}
-
-//***************************************************************************
-// UsgsAstroLsSensorModel::getImageSize
-//***************************************************************************
-csm::ImageVector UsgsAstroLsSensorModel::getImageSize() const {
-  return csm::ImageVector(m_nLines, m_nSamples);
-}
-
-//---------------------------------------------------------------------------

```

Comment:
Removed because this was commented-out, and getSensorModelState() was updated to getModelState() in the 3.0.3 version of the CSM API.